README.md

```diff
@@ -1,2 +1,45 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
+- [Homebrew](https://formulae.brew.sh/formula/node)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README. 
+2. Run ./bin/chs-dev modules enable strike-off-objections
+3. Run ./bin/chs-dev development enable strike-off-objections (this will allow you to make changes).
+4. Run docker using "tilt up" in the docker chs root directory.
+5. Open your browser and go to page http://chs.local/strike-off-obections
+
+These instructions are for a local docker environment.
+
+### Config variables
+
+- API_URL=http://<host:port> Required for accessing the backend application
```

Comment:
Most repos at CH use a table layout for this, for example: https://github.com/companieshouse/oracle-query-api#configuration

Reply:
Added a table